spring-integration-ip/src/main/java/org/springframework/integration/ip/dsl/UdpOutboundChannelAdapterSpec.java

```diff
@@ -102,12 +107,12 @@ public UdpOutboundChannelAdapterSpec ackCounter(int ackCounter) {
 	}
 
 	/**
-	 * @param socketExpressionFunction the socket expression.
+	 * @param socketFunction the socket function.
 	 * @return the spec.
 	 * @see UnicastSendingMessageHandler#setSocketExpression(org.springframework.expression.Expression)
 	 */
-	public UdpOutboundChannelAdapterSpec socketExpression(Function<Message<?>, ?> socketExpressionFunction) {
-		this.target.setSocketExpression(new FunctionExpression<>(socketExpressionFunction));
+	public UdpOutboundChannelAdapterSpec socketExpression(Function<Message<?>, DatagramSocket> socketFunction) {
```

Comment:
Do you agree with me that `socketFunction()` sounds better as a name for this kind of method ?